Sources/CSwiftV.swift

```diff
@@ -50,29 +50,29 @@ public class CSwiftV {
     }
 
     internal static func cellsFromString(rowString: String, separator: String = ",") -> [String] {
-        return CSwiftV.split(separator, string: rowString).map { element in
+        return CSwiftV.split(separator: separator, string: rowString).map { element in
             if let first = element.characters.first, let last = element.characters.last where first == "\"" && last == "\"" {
-                let range = element.startIndex.successor() ..< element.endIndex.predecessor()
-                return element[range]
+                let result = String(element.characters.dropFirst().dropLast())
+                return result
             }
             return element
         }
     }
 
     internal static func recordsFromString(string: String) -> [String] {
-        return CSwiftV.split("\n", string: string).filter { $0.isNotEmptyOrWhitespace }
+        return  CSwiftV.split(separator: "\n", string: string).filter { ($0 as String).isNotEmptyOrWhitespace }
```

Comment:
Not really necessary, could work simply as the original $0.isNotEmptyOrWhitespace